version_test.go

```diff
@@ -45,14 +45,13 @@ func (*suite) TestCompare(c *gc.C) {
 		{"2.0.0.0", "2.0.0.0", 0},
 		{"2.0.0.1", "2.0.0.0", 1},
 		{"2.0.1.10", "2.0.0.0", 1},
-		{"2.0-_0", "2.0-00", 1},
```

Comment:
this was removed because 2.0-00 is not actually a valid version, so it errors out (it has a zero length tag specified).  It used to work because the tag was getting parsed as "0"